source/uwp/Renderer/AdaptiveCardRenderer.vcxproj

```diff
@@ -44,7 +44,7 @@
     <AppContainerApplication>true</AppContainerApplication>
     <ApplicationType>Windows Store</ApplicationType>
     <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
-    <WindowsTargetPlatformMinVersion>10.0.15063.0</WindowsTargetPlatformMinVersion>
+    <WindowsTargetPlatformMinVersion>10.0.18362.0</WindowsTargetPlatformMinVersion>
```

Comment:
this doesn't sound right. Are you trying to set WindowsTargetPlatformVersion to 10.0.18362.0?

Reply:
We need to bump the minimum version to ingest WinUI. See here: https://github.com/Microsoft/microsoft-ui-xaml#version-support

That said, it looks like I should correct the change to be this:
```
<WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
<WindowsTargetPlatformMinVersion>10.0.15063.0</WindowsTargetPlatformMinVersion>
```